### ISSUE TITLE
Fix html template issues

### ISSUE THREAD

**Issue by btzr-io** [btzr-io/godot-aria#4]
- [x] Don't enforce aria-live = assert by default. Use polite instead.
- [x] Accessible progress bar #3 
- [x] Hide canvas unsupported message from screen readers.
- [x] Error status is not announced to the screen reader.
- [x] Prevent google chrome from requesting translation.

>   I'm getting a "your browser does not support the canvas tag" message in the splash screen.
